mev-relay-rs/src/relay.rs

```diff
@@ -411,51 +405,53 @@ impl Relay {
         Ok(())
     }
 
+    // NOTE: we assume that `auction_request` and `signed_submission` have been validated.
     fn insert_bid_if_greater(
         &self,
         auction_request: AuctionRequest,
-        execution_payload: ExecutionPayload,
+        signed_submission: &SignedBidSubmission,
         value: U256,
-        builder_public_key: BlsPublicKey,
     ) -> Result<(), Error> {
+        let builder_public_key = auction_request.public_key.clone();
         if let Some(bid) = self.get_auction_context(&auction_request) {
-            if bid.value > value {
+            if bid.value() > value {
                 info!(%auction_request, %builder_public_key, "block submission was not greater in value; ignoring");
```

Comment:
it looks like we just need this for this log, and it also looks like we can just grab this from the `bid` here